cbbr_build/version.env

```diff
@@ -0,0 +1,6 @@
+# CBBR versions for latest release
+VERSION=FY2024
```

Comment:
I think we could pin all the versions like we do with DevDB?

Reply:
done in [0d1904f](https://github.com/NYCPlanning/db-cbbr/pull/28/commits/0d1904fea6af341e749387fdab6579a57bd24c50)